internal/fs/inode/hns_dir_test.go

```diff
@@ -129,6 +129,27 @@ func (t *HNSDirTest) TestShouldReturnNilWhenGCSFolderNotFoundForInHNS() {
 	assert.Nil(t.T(), result)
 }
 
+func (t *HNSDirTest) TestLookUpChildShouldLookUpConflicting() {
```

Comment:
name is not clear. what do you mean by lookup conflicting?

Reply:
The name of the caller function is "lookupConflicting"

Reply:
I dont see anywhere in the test method the word lookupconflicting.

Reply:
It was calling function from below if condition.
```
if strings.HasSuffix(name, ConflictingFileNameSuffix) {
	return d.lookUpConflicting(ctx, name)
}
```

Anyways updated test name. Please check